content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -282,6 +282,7 @@ Implicit m-n relations:
 
 - Use a specific [convention for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations)
 - Do **not** require the `@relation` attribute unless you need to [disambiguate relations](/concepts/components/prisma-schema/relations#disambiguating-relations) (this is not specific to implicit n-m relations).
+- If they use an `@relation` attribute for disambiguation, you can still not use the `fields` and `references` arguments as these are implicit and fixed for implicit m-n relations. 
```

Comment:
The added line might actually be to loose still, and could be replaced with something like this depending on the result of https://prisma-company.slack.com/archives/C01TM7UM83Y/p1655918528824109

```suggestion
- Do **not** require the `@relation` attribute unless you need to [disambiguate relations](/concepts/components/prisma-schema/relations#disambiguating-relations) (this is not specific to implicit n-m relations). You can only use a nameless argument or the `name` argument in that case: `@relation("MyRelationTable")` or `@relation(name: "MyRelationTable")`. Other arguments like `fields`, `references`, `onDelete` or `onUpdate` are not possible as their values are implicit and fixed for implicit m-n relations. 
```

Reply:
See comment above